ibtk/include/ibtk/FEDataManager.h

```diff
@@ -890,6 +890,8 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
      */
     SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > d_hierarchy;
     int d_coarsest_ln = IBTK::invalid_level_number, d_finest_ln = IBTK::invalid_level_number;
+    using f_data_idx_map_key_type = std::tuple<int, int>;
+    std::map<f_data_idx_map_key_type, int> d_f_data_idx_map;
```

Comment:
It would be clearer to add a little enum here for cell-centered vs. face-centered. I don't know if SAMRAI has such a thing but its easier to understand than making one zero and the other one.

Reply:
Also, could you add some documentation for this member variable describing exactly why we are storing this cloned data?

Reply:
Sure.

Could something like [`type_index`](https://en.cppreference.com/w/cpp/types/type_index) work without having to introduce an `enum`?

Reply:
Yup.